src/components/Dropdown.tsx

```diff
@@ -0,0 +1,85 @@
+import { useState } from 'react';
+import classNames from 'classnames';
+import { Person } from '../types/Person';
+import { useDebounce } from '../customHooks/useDebounce';
+
+type DropdownProps = {
+  people: Person[],
+  onSelected: (person: Person) => void,
+};
+
+export const Dropdown: React.FC<DropdownProps> = ({ people, onSelected }) => {
+  const [query, setQuery] = useState<string>('');
+  const [isFocused, setIsFocused] = useState<boolean>(false);
+
+  const debouncedQuery = useDebounce(query, 300);
+
+  const handleInputChange: React.ChangeEventHandler<HTMLInputElement>
+    = (event) => {
+      setQuery(event.target.value);
+    };
+
+  let filteredPeople = people;
+
+  if (query !== '') {
```

Comment:
```suggestion
  if (!!query) {
```
pls see [falsy values](https://developer.mozilla.org/en-US/docs/Glossary/Falsy)